tests/integration/targets/user_info/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+  block:
+    - name: Retrieve info about the users
+      scale_computing.hypercore.user_info:
+      register: users
+    - ansible.builtin.assert:
+        that:
+          - users.records != []
+          - "{{ users.records[0].keys() | sort == ['fullname', 'roles', 'session_limit', 'username', 'uuid'] }}"
```

Comment:
```
(users.records[0].keys() | sort) == ['full_name', 'roles', 'session_limit', 'username', 'uuid']
```

Reply:
Tanks for the comment, integration test fixed